v2/azure-arm.yaml

```diff
@@ -1288,6 +1288,9 @@ objectModelConfiguration:
         $isResource: false
       DeepCreatedOriginProperties_STATUS:
         $isResource: false
+      DomainValidationProperties:
+        ValidationToken:
+          $isSecret: false # This might be a secret, @theunrepentantgeek is not sure
```

Comment:
Also not sure... we should find+ read their docs?

Reply:
It's not a conventional secret.

If you're interacting with a domain name provider, this is the key you provide to prove ownership of the domain.